src/academy/generics/ContainerUtility.java

```diff
@@ -0,0 +1,208 @@
+package academy.generics;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class ContainerUtility {
+
+	private static final String TEST_STRING = "WBR for Roma & inventorsoft";
+	private static final Long NUMBER_LONG = Long.valueOf(777L);
+	private static final Integer NUMBER_INTEGER = Integer.valueOf(555);
+	private static final Integer NUMBER_INTEGER_BAD = Integer.valueOf(666);
+
+	private static final List<Number> TEST_NUMBER_ODD_LIST = Arrays.asList(-3,1,3,5,7,9);
+	private static final List<Number> TEST_NUMBER_LIST = Arrays.asList(-100,-4f,-3,0,1,2,3,4d,5,6,7,8L,9,10);
+	private static final List<Integer> TEST_INTEGER_LIST = Arrays.asList(-100,-4,-3,0,1,2,3,4,5,6,7,8,9,10);
+	private static final List<Float> TEST_FLOAT_LIST = Arrays.asList(-2f, 1f, 2.2f, 4.0f, 100f);
+	private static final List<String> TEST_STRING_LIST1 = Arrays.asList("a0","a1","a2","a3","a4","a5");
+	private static final List<String> TEST_STRING_LIST2 = Arrays.asList("b0","b1","b2","b3","b4");
+	private static final int INDEX0 = 2, INDEX1 = 4, INDEX2 = 1;
+	private static final int BAD_INDEX1 = -1, BAD_INDEX2 = TEST_STRING_LIST2.size();
+
+	
+	// 3.
+	public static int countEvenNumbers(Collection<? extends Number> coll) {
+		return coll.stream()
+				.map( x -> x.doubleValue() )
+				.mapToInt( x -> x%2 == 0 ? 1 : 0 )
+				.sum();
+	}
+	
+	public static long countEvenNumbersLongResult(Collection<? extends Number> coll) {
```

Comment:
Is this method necessary here ?. As far as I can see in the task, there is no any information about  countEvenNumbersLongResult, countEvenNumbersWithoutDecimal.